### DISCUSSION TITLE
Can I use cache decorator in a module function and then use on fastapi main file?

### DISCUSSION THREAD

**Discussion by TheoOliveira** [long2ice/fastapi-cache#418] [General]
I have few functions that are in modules, i don't want to cache the entire fn route but the fn that actually get data. is it possible in case i use the decorator where the functions are declared and when run on the other file would work?

**Comment by Lservi** [comment]
I have the same question. Have you found any information on this matter?

**Reply by TheoOliveira** [reply]
Hey bro, The project i worked on i don't have access to the repo anymore but the way i did it was to create the modules and then on the main file i  created wrapper functions and use the cache decorator.
